src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/IncorporationController.java

```diff
@@ -97,16 +97,17 @@ public ResponseEntity<Object> getIncorporation(
         }
     }
 
-    @GetMapping("/costs")
+    @GetMapping("/{" + URL_PARAM_INCORPORATION_ID + "}/costs")
```

Comment:
It looks like this new end-point needs adding to the `schema.json` file too. @dlloyd1-ch ?